src/bullet/index.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components'
+import { color } from '_utils/branding'
+import Bullet from './Bullet'
+
+const StyledBullet = styled(Bullet)`
+  & {
+    box-sizing: content-box;
+    width: 6px;
+    height: 6px;
+    background-color: ${color.white};
+    border: 2px solid ${color.primaryText};
+    border-radius: 50%;
+  }
+
+  &.kirk-bullet--addon {
+    border-color: ${color.border};
+  }
+
+  &.kirk-bullet--small {
+    width: 4px;
+    height: 4px;
+  }
+
+  &.kirk-bullet--map {
```

Comment:
I'm not sure map is really clear here. As it will not be used in the map but in the itinerary component

Reply:
It's more a highlight WDYT? 🤷‍♂

Reply:
@mcellier actually requested to have a separate component so he could use it on maps too

Reply:
Yes, I did!